charts/fybrik/notebook-test-readflow.tls.values.yaml

```diff
@@ -38,24 +38,53 @@ katalogConnector:
     use_mtls: true
     certs:
       certSecretName: "test-tls-katalog-connector-certs"
-      certSecretNamespace: "fybrik-system"
       cacertSecretName: "test-tls-ca-certs"
-      cacertSecretNamespace: "fybrik-system"
 
 opaConnector:
+  serverURL: https://opa:443
   tls:
     use_tls: true
     use_mtls: true
     certs:
       certSecretName: "test-tls-opa-connector-certs"
-      certSecretNamespace: "fybrik-system"
       cacertSecretName: "test-tls-ca-certs"
-      cacertSecretNamespace: "fybrik-system"
   service:
     port: 8443
 
 opaServer:
 
+  podSecurityContext:
+    runAsNonRoot: false
+    runAsUser: 0
+
+  containerSecurityContext:
+    privileged: true
+    allowPrivilegeEscalation: true
+    capabilities:
+      add:
+        - NET_BIND_SERVICE
+
+  tls:
+    use_tls: true
```

Comment:
I am not sure about that. to turn on mtls the user should set the following flags in values.yaml otherwise he can omit them

```
args:
- "--authentication=tls"
- "--tls-ca-cert-file=/data/tls-cacert/ca.crt